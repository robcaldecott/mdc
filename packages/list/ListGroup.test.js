import React from "react";
import {
  ListGroup,
  ListGroupSubheader,
  ListGroupDivider,
  List,
  ListItem
} from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <ListGroup className="custom">
        <ListGroupSubheader>List 1</ListGroupSubheader>
        <List>
          <ListItem>Item 1</ListItem>
          <ListItem>Item 2</ListItem>
          <ListItem>Item 3</ListItem>
        </List>
        <ListGroupDivider />
        <ListGroupSubheader>List 2</ListGroupSubheader>
        <List>
          <ListItem>Item 1</ListItem>
          <ListItem>Item 2</ListItem>
          <ListItem>Item 3</ListItem>
        </List>
      </ListGroup>
    )
  ).toMatchSnapshot();
});
