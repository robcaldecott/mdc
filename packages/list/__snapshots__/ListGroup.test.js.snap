// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-list-group custom"
>
  <h3
    className="mdc-list-group__subheader"
  >
    List 1
  </h3>
  <ul
    className="mdc-list"
  >
    <li
      className="mdc-list-item"
    >
      Item 1
    </li>
    <li
      className="mdc-list-item"
    >
      Item 2
    </li>
    <li
      className="mdc-list-item"
    >
      Item 3
    </li>
  </ul>
  <hr
    className="mdc-list-divider"
    role="separator"
  />
  <h3
    className="mdc-list-group__subheader"
  >
    List 2
  </h3>
  <ul
    className="mdc-list"
  >
    <li
      className="mdc-list-item"
    >
      Item 1
    </li>
    <li
      className="mdc-list-item"
    >
      Item 2
    </li>
    <li
      className="mdc-list-item"
    >
      Item 3
    </li>
  </ul>
</div>
`;
